flow/ProtocolVersions.cmake

```diff
@@ -8,12 +8,12 @@
 # used and should not be changed from 0.
 #                                                          xyzdev
 #                                                          vvvv
-set(FDB_PV_DEFAULT_VERSION                      "0x0FDB00B072000000LL")
-set(FDB_PV_FUTURE_VERSION                       "0x0FDB00B073000000LL")
-set(FDB_PV_MIN_COMPATIBLE_VERSION               "0x0FDB00B071000000LL")
-set(FDB_PV_MIN_INVALID_VERSION                  "0x0FDB00B074000000LL")
-set(FDB_PV_LEFT_MOST_CHECK                      "0x0FDB00B100000000LL")
-set(FDB_PV_LSB_MASK                             "0xF0FFFFLL")
+set(FDB_PV_DEFAULT_VERSION                      "0x0FDB00C071300000LL")
+set(FDB_PV_FUTURE_VERSION                       "0x0FDB00C072000000LL")
+set(FDB_PV_MIN_COMPATIBLE_VERSION               "0x0FDB00C071200000LL")
+set(FDB_PV_MIN_INVALID_VERSION                  "0x0FDB00C073000000LL")
```

Comment:
My understanding is that FDB_PV_MIN_INVALID_VERSION should be higher than the intended protocol version of the next release. If we go for 8.0 then it should set it to 0x0FDB00C081000000LL. @sfc-gh-etschannen what do you think?

Reply:
IIUC, we haven't yet made a decision for the next protocol version. We'll have to make the decision within the next week or so, and we can update the versions here accordingly before we tag/ship the release, unless you think that'll cause issues.

Reply:
It is ok, we just need not to forget to change it later.